### DISCUSSION TITLE
I wish to make the widget/plugin store for Trilium.

### DISCUSSION THREAD

**Discussion by WantToLearnJapanese** [TriliumNext/Trilium#4603] [Ideas] [answered]
I have seen some posts discussing why Trilium is not gaining popularity while it has powerful potential.
One of the reasons, I think, is there is no convenient one-click store for Trilium.
Currently, I am not sure in which format the store should distribute plugins, but it won't be difficult to define such standards, I believe.

**Comment by meichthys** [comment, marked as answer]
Duplicate of: https://github.com/zadam/trilium/discussions/1873 
@WantToLearnJapanese Could you discuss this over there?

**Comment by meichthys** [comment]
Also relevenat: https://github.com/zadam/trilium/discussions/4236
